src/registry_schemas/schemas/continuation_in.json

```diff
@@ -24,28 +25,86 @@
                 "foreignJurisdiction": {
                     "type": "object",
                     "required": [
-                        "name",
+                        "country",
                         "identifier",
                         "legalName",
-                        "incorporationDate"
+                        "incorporationDate",
+                        "affidavitFileKey"
                     ],
                     "properties": {
-                        "name": {
+                        "country": {
+                            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/foreign_jurisdiction#/properties/country",
+                            "title": "The jurisdiction country name."
+                        },
+                        "region": {
+                            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/foreign_jurisdiction#/properties/region",
+                            "title": "The jurisdiction region name."
```

Comment:
As above, this is a 2-letter code, which can be:
- AB, SK, MB, etc (not including BC)
- "FD"
- one of the US states

Reply:
^^ This will need API validation. Please add to respective ticket if not already there.

Reply:
yeah! and its "FEDERAL" not "FD" https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/foreign_jurisdiction.json#L20

Reply:
Oh. It's "FD" in COLIN. I'll convert it when I receive the COLIN data.